scipy/spatial/_plotutils.py

```diff
@@ -153,6 +153,32 @@ def voronoi_plot_2d(vor, ax=None, **kw):
     -----
     Requires Matplotlib.
 
+    Examples
+    --------
+    Set of point:
+
+    >>> import numpy as np    
+    >>> import matplotlib.pyplot as plt
+    >>> points = np.random.rand(10,2) #random
+    >>> plt.scatter(points[:,0],point[:,1])
+    >>> plt.show()
+
+    Voronoi diagram of the points:
+
+    >>> from scipy.spatial import Voronoi, voronoi_plot_2d
+    >>> vor = Voronoi(points)
+
+    using voronoi_plot_2d for visualisation:
```

Comment:
Surround `voronoi_plot_2d` by single backticks (`` ` ``). Doing so makes the name a link, and otherwise it will interpret the underscores as restructuredtext.

Reply:
and same a few lines lower.